Workouts/HomeView.swift

```diff
@@ -52,6 +56,7 @@ struct HomeView: View {
                     }
                 }
             }
+            .listStyle(GroupedListStyle()).environment(\.horizontalSizeClass, .regular)
```

Comment:
@emilyCheroske Stole that new list/tableview style from Jeff and Justin's project. I think it looks pretty good!

Reply:
Dude it looks awesome!